content/issue-3/robjtede.md

```diff
@@ -1,23 +1,150 @@
+We are thrilled to present our third interview with **Rob** ([@robjtede]), the core maintainer of [actix-web](https://github.com/actix/actix-web) and [deps.rs](https://deps.rs/). In this interview, we talked about how he became involved with actix-web, what motivated him to take over the project after [@fafhrd91] quit, and his experience of maintaining popular Rust projects.
+
+Thanks to **Rob** for sharing his story with us. We hope you enjoy this interview!
+
+![](/static/issue-3/robjtede.jpeg)
+
+> Rob and a Saitama cosplay from One Punch Man, and his avatar also is Saitama.
+
 # Introduction
 
 **Introduce yourself and share a bit about your background with Rust. When did you start learning Rust, and what inspired you to do so?**
 
-# Actix-web and deps.rs
+```quote
+author="Rob"
+avatar="/static/avatar/robjtede.jpeg"
+content="""
+Hey folks. I'm Rob; [@robjtede](https://github.com/robjtede) on GitHub/Discord/Twitter/Reddit/etc. I'm a professional software engineer in the finance sector focussed on low latency distributed systems, mainly using microservice architectures. I did a masters degree in computer science at the University of Sheffield in the UK and moved to London to start my career after that.
+
+I found out about Rust at some point during university and started [practicing it during 2017's Advent of Code](https://github.com/robjtede/adventofcode2017) event. It was more or less just a curiosity at that point, as my main development was focussed on JavaScript and other frontend skills plus some Node.JS which I used in my undergraduate dissertation project.
+
+It wasn't until I started working (a frontend role) that I started using Rust more. I had suggested to the (small) team that part of the monolithic application we were creating could be split out. It was written in Java, and I had no intention of writing Java so Rust seemed like a good idea to inject into our stack. Most of the team had at least heard of Rust and understood its stated benefits so they gave me free license to develop this service alongside my normal frontend work.
+
+In the 4 years since then, I've transitioned to a purely backend role; as the team grew I needed to focus more on one side of the stack in order to keep my skills sharp. This lead to a desire to go deeper into Rust and its ecosystem.
+"""
+```
+
+# actix-web
```

Comment:
Can "Actix Web" always be formatted like this, please, ie. capitalized and no hyphen:

```suggestion
# Actix Web
```